sql/accounts/accounts.go

```diff
@@ -19,6 +20,8 @@ func load(db sql.Executor, address types.Address, query string, enc sql.Encoder)
 			stmt.ColumnBytes(3, account.TemplateAddress[:])
 			account.State = make([]byte, stmt.ColumnLen(4))
 			stmt.ColumnBytes(4, account.State)
+			// TODO: error handling here
+			codec.DecodeFrom(stmt.ColumnReader(5), &account.Storage)
```

Comment:
I think it's fine to panic if decoding fails as it means that the DB content is broken (it's an unrecoverable condition).
```suggestion
			_, err := codec.DecodeFrom(stmt.ColumnReader(5), &account.Storage)
			if err != nil {
				panic(fmt.Sprintf("decoding account storage: %v, err")
			}
```